modules/ROOT/pages/ts-cannot-process-B2B.adoc

```diff
@@ -0,0 +1,84 @@
+= Cannot process B2B transactions successfully
+
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You are performing tests on the flow or when your partner sends you B2B messages and you determine that the transactions are not getting processed.
+
+== Causes
+
+
+== Solutions
+
+=== Verify Flow configurations
+
+Verify your message flow configuration to ensure it matches the configuration per your partner.
+
+. From within your Anypoint Partner Manager environment (either Sandbox or Production) select your <Hostname> profile.
+. Select *Message Flows*.
+. Search for your message flow by the partner name or the message type such as 850.
+. Once you locate the message flow, click on the message flow card to open it up.
+. From the Message flow configurator, verify the following information per your agreement with your trading partner:
+
+** The correct identifiers are associated with the trading partner.
+
+** The appropriate receive endpoint is configured in the flow.
+
+** The appropriate document type (such as X12, version - 4010, message type - 850) is configured.
+
+** Verify that you have the appropriate acknowledgment configuration.
+
+** Verify that you have an appropriate DataWeave map in the transformation section.
+
+** Verify that you have configured an appropriate target such as both the message type of the target and the associated endpoint configuration.
+
++
+
+. If all of these look good, then ensure that the last deployment for this flow was successful by viewing the latest deployment history on the right side of the flow.
+
+. If you observe a Failed status, try to deploy the message flow to ensure all your configurations have persisted on the Mule runtime apps.
+
+NOTE: It can take a maximum interval of 5 minutes for your most recent changes to be effective on the runtime B2B Mule apps.
```

Comment:
I think this information belongs elsewhere, unless you are referring only to the final step on this operation.